### ISSUE TITLE
Clarification about the UserName, Password, and HostName binding properties

### ISSUE THREAD

**Issue by gabesmsft** [Azure/azure-functions-rabbitmq-extension#141]
It would be helpful to make some clarifications about the UserName, Password, and HostName binding properties in the following doc:
[https://github.com/Azure/azure-functions-rabbitmq-extension/](https://github.com/Azure/azure-functions-rabbitmq-extension/)

For example, the UserName and Password properties need to reference app settings instead of directly referencing values or using %%. Eg: **UserName = "UserNameAppSetting"** is the correct format, whereas UserName = "%UserNameAppSetting%" or UserName = "MyValidUserName" will cause the Functions host to hit the following error at startup:
Microsoft.Azure.WebJobs.Host: Unable to resolve the value for property 'RabbitMQAttribute.UserName'. Make sure the setting exists and has a valid value.

Since HostName needs to be a direct value or use %% to reference an app setting, it would be particularly helpful to make the distinction about how UserName and Password work to avoid confusion.

Assuming these three properties are only intended for Output bindings, it would be good to call that out in the doc as well. Although I see a HostName property for Triggers, I don't see the UserName or Password properties for triggers. I do see the UserNameSetting and PasswordSetting properties for Triggers, but neither these properties appear to be read when using a Trigger.

TLDR:
I have the following, whichworks as expected in my Output binding:
binding properties:
` HostName = "%HostNameAppSetting%", UserName = "UserNameAppSetting", Password = "PasswordAppSetting",

```
local.settings.json:
    "HostNameAppSetting": "MyValidHostName",
    "UserNameAppSetting": "MyValidUserName",
    "PasswordAppSetting": "MyValidPassword"
```
	
However, the PasswordSetting or UserNameSetting bindings properties (which are present in place of Password and UserName) don't appear to bet honored in my Trigger. Eg:
`HostName = "%HostNameAppSetting%", UserNameSetting = "UserNameAppSetting", PasswordSetting ="PasswordAppSetting",
`
Am using extension version 1.0.0-beta

**Comment by gabesmsft** [comment]
Update about using HostName, UserNameSetting, and PasswordSetting in a Trigger binding:
These will work if you do either of the following:
a) Hard-code the values. Eg:
HostName = "MyValidHostName", UserNameSetting = "MyValidUserName", PasswordSetting ="MyValidPassword"
b) Use %% around the app setting name references. Eg:
HostName = "%HostNameAppSetting%", UserNameSetting = "%UserNameAppSetting%", PasswordSetting ="%PasswordAppSetting%"

**Comment by gabesmsft** [comment]
Additional note: I notice that ConnectionStringSetting takes precedence over HostName, etc, so might be one more thing to call out in the doc.

**Comment by gabesmsft** [comment]
[docs](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-rabbitmq) are updated with further info about the binding properties
